dotnet-skeleton-app/Controllers/SmsController.cs

```diff
@@ -4,7 +4,10 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
-using Nexmo.Api;
+using Nexmo.Api.Utility;
```

Comment:
Did this not get moved, or just accidentally used instead of the Vonage equivalent?